### PR TITLE
Improve localization guide

### PR DESCRIPTION
**Pull request by seokho-son**
This PR improves localization guide.

 - Introduce two parts
   - Initiate a new localization team
   - Get involved in an existing localization team
 - More examples
 - Note regarding permission
 - ...

### CODE REVIEW

**Comment by seokho-son** [comment]
## Preview for review

https://github.com/cncf/glossary/blob/38bf7501dc9aea02696a1a6e8d48b74074c670c1/LOCALIZATION.md

File: LOCALIZATION.md [hunk 1]
```diff
@@ -3,25 +3,53 @@
...
 
 Through this issue, the new localization team will
+ - get a development branch for the localization works (ex: [dev-ko](https://github.com/cncf/glossary/tree/dev-ko) branch). 
  - get a permission for approving localization contents.
- - get a development branch for the localization works. (use the development branch for following localization tasks)
+   ```
+   The approvers of l10n team will have a push permission to this repository.
+   It is to make l10n approvers manage (merge) PRs for l10n contributions in your development branch.
+   Merging a PR to the `main` branch by l10n approvers is restricted.
+   Even if they are possible to review a PR to the `main` branch and give an approval to the PR, they should not approve ths PR.
```
Comment (seokho-son) [inline LOCALIZATION.md:39]:
```suggestion
   Even if they are possible to review a PR to the `main` branch and give an approval to the PR, they should not approve the PR. 
```

File: LOCALIZATION.md [hunk 2]
```diff
@@ -44,39 +70,61 @@ Find existing `[languages]` block in `config.toml` and add a block for the new l
...
+
+### 4. Merging the localization branch into `main` branch
+
+After completing the minimum required terms for a new localization, open a PR to merge commits in the localization branch into the `main` branch. (ex: `dev-ko` to `main`).
+
+Once the PR is merged, the localized content will go live on its website 🎉
+
+---
+
+# Join an existing localization team
```
Comment (CathPag) [inline LOCALIZATION.md:125]: Updated this section. I don't think we need a lot more. If there is a team already, they should be able to onboard the newbie, what do you think?

File: LOCALIZATION.md [hunk 3]
```diff
@@ -3,25 +3,51 @@
...
+
+- The Glossary project does not have a localization team for your language yet
+  - check the language list from [localization progress](https://github.com/cncf/glossary/projects/2) and ask in the [glossary-localizations channel in Slack](https://cloud-native.slack.com/archives/C02N2RGFXDF). There may be a group of volunteers who haven't gotten started yet.
 - At least three volunteers
 - At least two approvers 
-  - Each term must be approved by two approvers
-  - Unanimous consent must be reached to be approved 
+  - two approvers must approve each term
+  - unanimous consent must be reached to be approved 
+  - at least one volunteer must have basic GitHub knowledge and be willing to help the others get up to speed (such as PR, review process, branch, ..)
```
Comment (CathPag) [inline LOCALIZATION.md:23]: I think this should be at least one, not all. But this person should be willing to help others.

**Review by CathPag — COMMENTED** [review]
I think this looks great! I did a few edits and added two things that are highlighted in the comment section. What do you think? 

There is one sentence I don't understand. If we address that, we are ready to merge as far as I'm concerned.

File: LOCALIZATION.md [hunk 4]
```diff
@@ -3,25 +3,51 @@
...
 
 ## Instructions to initiate a new localization
 
-### Open an Issue to notify initiation of a new localization 
+### 1. Opening an issue to notify initiation of a new localization
```
Comment (CathPag) [inline LOCALIZATION.md:27]: We should also mention joining Slack. I think that may be the first step, what do you think?
****

# 1. Joining the Slack channel and saying hi

If you're interested in creating a new localization team, whether you have a team or are on your own, please join the #glossary-localizations channel on the CNCF slack and say hi. If you have at least three volunteers and are ready to start, request a localization channel for your language (e.g., #glossary-localization-korean). If you are still looking for more volunteers, let everyone in the channel know. There may be someone in there looking for a team as well.

File: LOCALIZATION.md [hunk 5]
```diff
@@ -3,25 +3,53 @@
...
+
+- Glossary project does not have a localization team for your language yet
+  - check language list from [localization progress](https://github.com/cncf/glossary/projects/2) or ask it to [glossary-localizations channel in Slack](https://cloud-native.slack.com/archives/C02N2RGFXDF)
 - At least three volunteers
 - At least two approvers 
-  - Each term must be approved by two approvers
-  - Unanimous consent must be reached to be approved 
+  - each term must be approved by two approvers
+  - unanimous consent must be reached to be approved 
+  - candidates need to have basic knowledge in Github (such as PR, review process, branch, ..)
```
[the remark this answers is not in the document]
Comment (seokho-son) [reply LOCALIZATION.md]: Looks better :) Thanks!

File: LOCALIZATION.md (hunk 2 above)
Comment (seokho-son) [reply LOCALIZATION.md:125]: Looks good to me :)

File: LOCALIZATION.md [hunk 6]
```diff
@@ -3,25 +3,51 @@
...
 
-### Add a new language setting to site configuration
+Open a PR with the localization initiation following this example: https://github.com/cncf/glossary/pull/291.
+
+
+
+#### 2-1. Adding a new language setting to site configuration
 
 New languages need to be set up inside of `config.toml`.
 Find existing `[languages]` block in `config.toml` and add a block for the new language below other language blocks.
```
Comment (seokho-son) [inline LOCALIZATION.md:53]:
```suggestion
To add a new language to site, you needs to modify [config.toml](https://github.com/cncf/glossary/blob/main/config.toml#L54) (Note: localization teams should use their assigned development branch for this task).
`[languages]` block of `config.toml` is used to configure languages for the site. `[languages.en]` means language configuration for English and `[languages.ko]` means language configuration for Korean.
Go to `[languages]` block in `config.toml` and add a new block like `[languages.ko]` with a language-specific configuration.
For instance, Korean localization team can add `[languages.ko]` block after `[languages.en]` block.
```
How about this ? :)

File: LOCALIZATION.md [hunk 7]
```diff
@@ -3,25 +3,53 @@
...
 
-### Add a new language setting to site configuration
+Fulfill following tasks and open a PR. (ex: https://github.com/cncf/glossary/pull/291)
+
+
+
+#### 2-1. Add a new language setting to site configuration
 
 New languages need to be set up inside of `config.toml`.
 Find existing `[languages]` block in `config.toml` and add a block for the new language below other language blocks.
```
[the remark this answers is not in the document]
Comment (seokho-son) [reply LOCALIZATION.md:53]: https://github.com/cncf/glossary/pull/341#discussion_r790303902

File: LOCALIZATION.md (hunk 6 above)
Comment (CathPag) [reply LOCALIZATION.md:57]: Looks good. I made a few edits. Can you verify, it's all correct. If so, let's commit.

**Review by CathPag — APPROVED** [review]
If my edits for the suggested changes work, this is ready I think

**Review by jihoon-seo — APPROVED** [review]
`Approve test` according to a request from @seokho-son 😊

Note on test result:
As a triage, approval from @jihoon-seo didn't work. (as expected)

File: LOCALIZATION.md [hunk 8]
```diff
@@ -3,28 +3,59 @@
...
+If you're interested in creating a new localization team, whether you have a team or are on your own, please join the #glossary-localizations channel on the CNCF slack and say hi. If you have at least three volunteers and are ready to start, request a localization channel for your language (e.g., #glossary-localization-korean). If you are still looking for more volunteers, let everyone in the channel know. There may be someone in there looking for a team as well. 
+
+### 2. Opening an issue to notify initiation of a new localization 
+
+[Open an issue](https://github.com/cncf/glossary/issues/new/choose) 
+and select the `Initiate a New Localization Team` template.
+
+With this issue, new localization teams are requesting:
+ - their own development branch (ex: [dev-ko](https://github.com/cncf/glossary/tree/dev-ko) branch). 
+ - permission for approving localized content
```
Comment (seokho-son) [inline LOCALIZATION.md:38]:
```suggestion
 - permission for approving localized content
   - based on discussions, a l10n team needs to select suitable candidates for l10n content approvers (up to 4 candidates)
```
@CathPag 
How about adding this sentence to guide the maximum approvers for each team :)

**Review by seokho-son — COMMENTED** [review]
Update based on a maintainers' meeting.

**Review by CathPag — APPROVED** [review]
Looks great! Let's get this merged today if possible

**Review by JasonMorgan — APPROVED** [review]
LGTM

**Pull request merged**